### PR TITLE
chore(ci): update dependencies and fix file upload test

### PR DESCRIPTION
**Pull request by anehx**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,35 +30,35 @@
...
+    "ember-changeset": "^3.14.1",
+    "ember-changeset-validations": "^3.15.2",
     "ember-cli-babel": "^7.26.6",
     "ember-cli-htmlbars": "^5.7.1",
     "ember-cli-sass": "^10.0.0",
     "ember-cli-showdown": "^4.5.0",
     "ember-cli-string-helpers": "^6.1.0",
     "ember-composable-helpers": "^4.5.0",
-    "ember-concurrency": "^2.1.0",
+    "ember-concurrency": "^2.0.2",
```
Comment (czosel) [inline package.json:50]: why downgrade?

**Review by czosel — APPROVED** [review]
Otherwise LGTM

File: package.json (hunk 1 above)
Comment (anehx) [reply package.json]: Missed a 2, fixed

**Pull request merged**